Conch/lib/Conch/Validation.pm

```diff
@@ -622,6 +644,23 @@ sub clear_results ( $self ) {
 	return $self;
 }
 
+# Internal error representation. not used outside of this module. Extends
+# 'Mojo::Exception' to store a 'hint' attribute.
+package Conch::Validation::Error;
```

Comment:
Please break this out into a separate file

Reply:
Honest question. I intended fully for this exception to only be used within Conch::Validation and this was the best approximation of a 'private class' as you'd find in other languages. Is there a particular reason to do this in Perl or is it a preference?

Reply:
Modules in their own file is convention, yes, but it's a sensible one because it makes finding code easier. When working with the code, I will expect to find Conch::Validation::Error in lib/Conch/Validation/Error.pm and will waste time hunting for it otherwise.